src/modules/formBuilder/components/itemEditor/pickLists/PickListOption.tsx

```diff
@@ -0,0 +1,111 @@
+import ControlledTextInput from '@/modules/form/components/rhf/ControlledTextInput';
+import { FormItemControl } from '@/modules/formBuilder/components/itemEditor/types';
+import { Component } from '@/types/gqlTypes';
+
+interface PickListOptionProps {
+  control: FormItemControl;
+  index: number;
+  formItemComponent: Component;
+  setPickListInitialSelected: (code: string) => void;
+  isCodeUnique: (coe: string) => boolean;
+  code?: string;
+}
+
+const PickListOption: React.FC<PickListOptionProps> = ({
+  control,
+  code,
+  index,
+  formItemComponent,
+  isCodeUnique,
+  setPickListInitialSelected,
+}) => {
+  const onChangeInitialSelected = useCallback(
+    (value: boolean) => {
+      // Even though it's a controlled RHF checkbox, we still provide an onChange callback
+      // in order to set all the other pick list options' `initialSelected` values to false when this one is set to true.
+      if (value && code) setPickListInitialSelected(code);
+    },
+    [setPickListInitialSelected, code]
+  );
+
+  return (
+    <Stack gap={2}>
+      <ControlledTextInput
+        control={control}
+        name={`pickListOptions.${index}.code`}
+        label='Code'
+        helperText='Must be unique'
+        required={true}
+        rules={{
+          validate: (input) => isCodeUnique(input),
+        }}
+      />
+      <ControlledTextInput
+        control={control}
+        name={`pickListOptions.${index}.label`}
+        label='Label'
+        helperText='If Label is not provided, Code will be displayed'
+      />
+
+      <ControlledCheckbox
+        name={`pickListOptions.${index}.initialSelected`}
+        control={control}
+        label='Initially selected'
+        helperText='Whether this choice is selected by default'
+        onChange={onChangeInitialSelected}
+      />
+
+      {formItemComponent !== Component.Dropdown && (
+        // Helper text is only supported for radio/checkbox, not dropdown options
+        <ControlledTextInput
+          control={control}
+          name={`pickListOptions.${index}.helperText`}
+          label='Helper text'
+          helperText='Helper text, including html'
+        />
+      )}
+
+      {formItemComponent === Component.Dropdown && (
+        // Secondary label and group are only supported for dropdowns
+        <>
+          <ControlledTextInput
+            control={control}
+            name={`pickListOptions.${index}.secondaryLabel`}
+            label='Secondary Label'
+            helperText='Secondary label for dropdown items'
+          />
+
+          <Stack direction='row' gap={1}>
+            <ControlledTextInput
+              control={control}
+              name={`pickListOptions.${index}.groupLabel`}
+              label='Group Label'
+              helperText='Label for group that this choice belongs to, if grouped'
+            />
+            <ControlledTextInput
+              control={control}
+              name={`pickListOptions.${index}.groupCode`}
```

Comment:
I think that it would be nice for v2 to provide a better UI for managing choice groups in the UI, maybe with some help from design first. For v1, I think this is ok.

It looks like we use groups for 2 purposes:
- in dropdowns, we visually arrange choices under groups
- we can use the groupCode in conditional logic like enableWhen, but it looks like that's not currently turned on in the form builder because it has only one use case? https://github.com/greenriver/hmis-frontend/blob/c7482bfacf0ec245c06ebcd1a8dd4f55cb1294fd/src/modules/formBuilder/components/itemEditor/conditionals/EnableWhenCondition.tsx#L77 Does this mean we should disable groupCode here too?

Reply:
Yeah, I think this is good for v1! There is only 1 specific use case for conditional based on the group code (prior living situation). However, I could imagine plenty of situations where you would want to visually group the choices to separate them into groups. So, yes, I think we should support it here.

Reply:
Sounds good, yes I definitely think `groupLabel` should be supported for v1. I was asking here specifically about `groupCode`, which isn't currently supported in the form builder's EnableWhen manager. (it seems to me that `groupCode` and `groupLabel` are operating independently of one another.) However, I'm happy to leave both if you prefer.

Reply:
Oh, I see. leaving both seems fine for this first pass. Thanks!